libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -39,6 +39,14 @@ bytes toBytes(uint8_t _b)
 	return bytes(1, _b);
 }
 
+enum class Flags: uint8_t
+{
+	Immutable = 0,
+	Mutable = 1,
```

Comment:
These are not flags, but actual values in the spec: "mut = const | var"

Reply:
Well, yeah but the spec still calls them flags :)

https://webassembly.github.io/spec/core/binary/types.html#global-types
> Global types are encoded by their value type and a flag for their mutability.

As with the maximum field, I'm fine with it either way. I can change it something like this if you prefer:

``` c++
enum class Mutability: uint8_t
{
	Const = 0,
	Var = 1,
}
```

Reply:
Actually, might have been better to call it `FlagValues` because `Flags` looks a bit as if they were their names/masks (though repeating values would make it clear that they're not).

Reply:
Replaced with `Mutability` enum.

Reply:
> Well, yeah but the spec still calls them flags :)

I am looking at a different section and probably a different version of the spec, where it is not called flags :)